scripts/ci/common.psm1

```diff
@@ -0,0 +1,110 @@
+<#
+.SYNOPSIS
+    Given the path to the list of raw git changes, returns an array of
+    those changes rooted in the git root directory.
+.DESCRIPTION
+    For example, the raw git changes will contain lines like:
+
+    Assets/File.cs
+
+    This function will return a list of paths that look like (assuming
+    that RepoRoot is C:\repo):
+
+    C:\repo\Assets\File.cs
+#>
+function GetChangedFiles { 
```

Comment:
Couldn't we just write the full path when writing the file in githubchanges.ps1? The root path is already being joined to the file path there when we check if the changed file still exist:
```powershell
$joinedPath = Join-Path -Path $RepoRoot -ChildPath $changedFile
```

Reply:
I thought about this when I was writing this and opted to only save the relative path because of the reasoning here:

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/778d323aaf43c2ed253be76bf58b29c019bd30d2/scripts/ci/githubchanges.ps1#L79

However, this is probably a case of overengineering - I think I only did some of this because there was some recent chatter about trying to parallelize some of our pipelines, in which case producing artifacts that could be somehow shared/information shared between places would be useful.

Just to reduce the chance I break something because I know things are working now, I think that I'll leave this as-is, but I will eat my proverbial shoe if we hit issues with this going forward and I can change it up if it ends up being unwieldy.